From 4abf223be56e854ce99f4e27eb1915856f1e2d4c Mon Sep 17 00:00:00 2001
From: Ben Peled <bpeled@marvell.com>
Date: Wed, 19 May 2021 19:38:58 +0300
Subject: [PATCH 1566/1921] dts: cn9131/2: enable crypto

add second and third crypto to cn9131/2

Signed-off-by: Ben Peled <bpeled@marvell.com>
Change-Id: I0ffcce1034d77e0b64442b6bfdc8f7cdd828f01c
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/52170
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Kostya Porotchkin <kostap@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9131-db.dtsi | 2 +-
 arch/arm64/boot/dts/marvell/cn9132-db.dtsi | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9131-db.dtsi b/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
index dfa360a598a3..cc93f58a67e9 100644
--- a/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
@@ -77,7 +77,7 @@
 #undef CP11X_PCIE2_BASE
 
 &cp1_crypto {
-	status = "disabled";
+	status = "okay";
 };
 
 &cp1_ethernet {
diff --git a/arch/arm64/boot/dts/marvell/cn9132-db.dtsi b/arch/arm64/boot/dts/marvell/cn9132-db.dtsi
index 20fb4ad0de03..2828c233e883 100644
--- a/arch/arm64/boot/dts/marvell/cn9132-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9132-db.dtsi
@@ -95,7 +95,7 @@
 #undef CP11X_PCIE2_BASE
 
 &cp2_crypto {
-	status = "disabled";
+	status = "okay";
 };
 
 &cp2_ethernet {
-- 
2.31.1

